apps/frontend/static_src/scss/components/toc.scss

```diff
@@ -0,0 +1,38 @@
+.toc {
+    margin-bottom: ($gutter * 2);
+
+    @include media-query(large) {
+        margin-bottom: ($gutter * 3);
+    }
+
+    &__heading {
+        font-size: 16px;
+        line-height: 1.3;
+        margin-bottom: $gutter;
+    }
+
+    &__content {
+        font-size: 18px;
+
+        ul {
+            list-style-type: none;
+            padding: 0;
+            margin: 0;
+        }
+
+        li {
+            display: flex;
+            align-items: center;
+            margin-bottom: ($gutter * 0.25);
+
+            &::before {
+                content: "";
+                display: inline-block;
+                height: 22px;
+                width: 22px;
+                margin-right: 10px;
+                background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj48cGF0aCBmaWxsPSIjMDA3RDdFIiBkPSJNMjgwLjIgMTUwLjJjLTcuMS02LjQtMTguMS04LTI1LjktNC4xcy0xNS4yIDEyLjQtMTUuMiAyMWwuMDAyIDU2TDE1MiAyMjRjLTEzLjIgMC0yNCAxMC44LTI0IDI0djE2YzAgMTMuMyAxMC44IDI0IDI0IDI0bDg4LS45djU2YzAgOS41MzEgNS42NTYgMTguMTYgMTQuMzggMjJhMjQuMDI1IDI0LjAyNSAwIDAgMCAyNS45MS00LjM3NWw5Ni04OC43NUMzODEuMiAyNjguMyAzODQgMjYxLjkgMzg0IDI1NS4yYy0uMzEzLTcuNzgxLTIuODc1LTEzLjI1LTcuODQ0LTE3Ljc1TDI4MC4yIDE1MC4yek0yNTYgMEMxMTQuNiAwIDAgMTE0LjYgMCAyNTZzMTE0LjYgMjU2IDI1NiAyNTYgMjU2LTExNC42IDI1Ni0yNTZTMzk3LjQgMCAyNTYgMHptMCA0NjRjLTExNC43IDAtMjA4LTkzLjMxLTIwOC0yMDhTMTQxLjMgNDggMjU2IDQ4czIwOCA5My4zMSAyMDggMjA4LTkzLjMgMjA4LTIwOCAyMDh6Ii8+PC9zdmc+");
```

Comment:
For a project like this I think it’s important we don’t hard-code base64 icons like this, which will make it hard for us to recover the actual image if we ever need to.

If the tooling isn’t doing this currently and it feels like a waste of time, just add a comment to explain which icon this is / which SVG file, and we’ll open an issue to set this up properly.

Reply:
Done in bootstrap eval